homeassistant/components/kef/media_player.py

```diff
@@ -298,11 +298,6 @@ def source_list(self):
         """List of available input sources."""
         return self._sources
 
-    @property
-    def available(self):
-        """Return if the speaker is reachable online."""
-        return self._is_online
-
     @property
     def unique_id(self):
```

Comment:
This is also a a provided `attr` now see https://github.com/home-assistant/core/blob/bbf77ed46f4380bfc7c0e228a46a9f8d6ca11cf1/homeassistant/helpers/entity.py#L293

`self._attr_unique_id` can be used!

Reply:
Done, including a few more...

Reply:
Top!